test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests.csproj

```diff
@@ -1,8 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
-    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net452;net46</TargetFrameworks>
+    <TargetFrameworks Condition="$(TARGET_FRAMEWORK) == ''">netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(TARGET_FRAMEWORK) == '' AND $(OS) == 'Windows_NT'">$(TargetFrameworks);net452;net46</TargetFrameworks>
+    <TargetFrameworks Condition="$(TARGET_FRAMEWORK) != ''">$(TARGET_FRAMEWORK)</TargetFrameworks>
```

Comment:
This third line seems kind of redundant :shrug:

Reply:
Maybe not. This is basically allowing $(TARGET_FRAMEWORK) to override what is in csproj? So you can pass it on the command-line?

Reply:
Correct, `TARGET_FRAMEWORK` is a property that is set when building the project for running the integration tests

https://github.com/open-telemetry/opentelemetry-dotnet/blob/808ae2f2bf0a47aaab90db75591b8b907d70b479/test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/Dockerfile#L12